src/app/body/create-new-session/create-new-session.component.ts

```diff
@@ -109,15 +109,16 @@ export class CreateNewSessionComponent implements OnInit {
       console.log("Successfully created the task");
       console.log(result);
       this.cloneTask.resetTask();
-      this.router.navigate(['/']);
+      this.router.navigate(['MyDashboard']);
     } catch (error) {
       this.errorHandlerService.getErrorCode(this.componentName, "InternalError");
       this.enableLoader = false;
     }
   }
 
   backToDashboard() {
-    this.location.back()
+    this.router.navigate(['MyDashboard']);
+    //this.location.back()
```

Comment:
I dont think this change is required in both create sprint and create new task

Reply:
I removed those lines from the file.